components/Navbar.jsx

```diff
@@ -1,31 +1,77 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import Link from "next/link";
 import Image from "next/image";
-import styles from "../styles/Navbar.module.css";
 import logo from "../public/logo.png";
+import styles from "../styles/Navbar.module.css";
+import { AiOutlineClose, AiOutlineMenu } from "react-icons/ai";
```

Comment:
I don't think using a library only for these icons is worth, please use a svg from other place instead